aten/src/ATen/native/cudnn/RNN.cpp

```diff
@@ -288,8 +332,15 @@ namespace {
       return batch_sizes.size() != 0;
     }
 
-    void set(IntArrayRef input_sizes, IntArrayRef batch_sizes_, bool batch_first) {
+#if defined(CUDNN_VERSION) && CUDNN_VERSION >= RNNV8VERSION
+    int batch_first = -1;
```

Comment:
Is this a global variable?!

Reply:
It was hiding as part of the struct but the declaration location makes it look global ;)

Removed as this was a vestigial variable from when I thought we needed to track the layout for creation of the `RNNDataDescriptor`s.